src/main/kotlin/com/ajaxproject/telegrambot/bot/service/FinanceService.kt

```diff
@@ -0,0 +1,44 @@
+package com.ajaxproject.telegrambot.bot.service
+
+import com.ajaxproject.telegrambot.bot.models.MongoExpense
+import com.ajaxproject.telegrambot.bot.models.MongoIncome
+import com.ajaxproject.telegrambot.bot.repository.FinanceRepositoryImpl
+import org.bson.types.ObjectId
+import org.springframework.stereotype.Service
+
+@Service
+class FinanceService(
+    private val incomeRepositoryImpl: FinanceRepositoryImpl<MongoIncome>,
+    private val expenseRepositoryImpl: FinanceRepositoryImpl<MongoExpense>,
```

Comment:
such repository usage leads to thoughts if you really need two separate mongo collection or you can combine them into one

Reply:
Okay, I'll try to rewrite it